ui/v2.5/src/components/Images/ImageDetails/Image.tsx

```diff
@@ -10,7 +10,6 @@ import {
 import { ErrorMessage, LoadingIndicator, Icon } from "src/components/Shared";
 import { useToast } from "src/hooks";
 import { TextUtils } from "src/utils";
-import * as Mousetrap from "mousetrap";
```

Comment:
I've actually been adding in regular mousetrap imports in #914 since Typescript 4.0 complains otherwise. There's no reason not to import them, MousetrapPause will always run when the page is loaded and affects every import.